src/components/TileCard/index.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { FC } from 'react';
+import classNames from 'classnames';
+
+export interface TileCardProps {
+  title: string;
+  link: string;
+  customLink?: {
+    component: any;
+    hrefProperty: string;
+  };
+  icon: FC<any>;
+  iconClassName?: string;
+  disabled?: boolean;
+}
+
+export const TileCard: React.FC<TileCardProps> = ({
+  title,
+  link,
+  icon,
+  disabled,
+  customLink = {
+    component: (props: any) => <a {...props} />,
+    hrefProperty: 'href',
+  },
+  iconClassName,
+}) => {
+  const Icon = icon;
+
+  return (
+    <div
+      className={classNames(
+        disabled
+          ? 'bg-gray-200 hover:cursor-not-allowed pointer-events-none shadow-none opacity-20'
+          : '',
```

Comment:
'bg-white focus-within:ring-2 focus-within:ring-inset focus-within:ring-indigo-500'

replace the empty quotes with this and remove those from below